cmd/kube-apiserver/app/server.go

```diff
@@ -382,7 +382,7 @@ func buildGenericConfig(
 		return
 	}
 	if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.APIServerTracing) {
-		if lastErr = s.Traces.ApplyTo(genericConfig.EgressSelector, genericConfig); lastErr != nil {
+		if _, lastErr = s.Traces.ApplyTo(genericConfig.EgressSelector, genericConfig); lastErr != nil {
 			return
 		}
```

Comment:
should this be setting `genericConfig.LoopbackClientConfig.Wrap` below?

Reply:
I am wrapping LoopbackClientConfig within Traces.ApplyTo, since it is common between staging/src/k8s.io/apiserver/pkg/server/options/recommended.go and cmd/kube-apiserver/app/server.go.  I could instead move it out if there is a place it would make more sense in.